### DISCUSSION TITLE
implementation

### DISCUSSION THREAD

**Discussion by btwbtw01** [Xilinx/finn#550] [Q&A]
Hi, everyone
     Can I implement VGG16, mobilenet and resnet20 on ZCU104 with 8 bits activations and weights? I try to implement it but I have problems when I try to transform model to onnx. The questions I have posted before https://github.com/Xilinx/finn/discussions/514
Thanks!!!

**Comment by nhma20** [comment]
Here are some links to examples of VGG and mobilenet implemented with Brevitas:
https://github.com/Xilinx/brevitas/tree/master/src/brevitas_examples/imagenet_classification/models
https://github.com/MinahilRaza/Brevitas_Fixed_Point/tree/master/training_scripts/models

Hope that is helpful
